src/tmp_dir.cpp

```diff
@@ -57,6 +57,10 @@ TmpDir::TmpDir(std::string prefix)
 
 void TmpDir::Execute(std::string exe, std::string args)
 {
+    if(miopen::IsEnabled(MIOPEN_DEBUG_SAVE_TEMP_DIR{}))
+    {
+        MIOPEN_LOG_I2(this->path.string());
```

Comment:
should we use a more descriptive output, like "temporary directory: " + this->path.string() ?

Reply:
will do